.gitlab-ci.yml

```diff
@@ -6,58 +6,21 @@ stages:
   - build
   - release
 
-# Fallback to support `includes`
-cache:
-  key: $CI_COMMIT_REF_SLUG
-  paths:
-    - node_modules/
-    - lib/
-
-.shared:
-  artifacts:
-    paths:
-      - lib/
-      - package.json
-      - CHANGELOG.md
-    expire_in: 30 days
-  cache:
-    key: $CI_COMMIT_REF_SLUG
-    paths:
-      - node_modules/
-      - lib/
-
-build_npm_package:
-  stage: build
-  extends: .shared
-  except:
-    refs:
-      - schedules
-      - tags
-    variables:
-      - $CI_COMMIT_MESSAGE =~ /\[ci-release\]/
-  script:
-    - yarn build
-  cache:
-    policy: pull
-
 release:
   stage: release
-  extends: .shared
-  cache:
-    policy: pull
-  artifacts: {}
+  extends:
+    - .except-ci-release
+    - .cache-pull-only
   only:
     - master
-  except:
-    refs:
-      - schedules
-      - tags
-    variables:
-      - $CI_COMMIT_MESSAGE =~ /\[ci-release\]/
+  artifacts:
+    paths:
+      - CHANGELOG.md
```

Comment:
sorry, i keep forgetting. why do we need this artifact here?

Reply:
No, valid question. Since it's the last stage in this pipeline, we can remove it actually 👍

Reply:
updated

Reply:
great, i think the same comment then also applies to https://git.brickblock.sh/devops/project-templates/raw/master/gitlab-ci/release.yml (just as a sidenote)